primitives/src/sybil.rs

```diff
@@ -1,34 +1,100 @@
 use codec::{Decode, Encode};
+use fixed::traits::Fixed;
 use rstd::vec::Vec;
-use sp_core::RuntimeDebug;
+use sp_core::{RuntimeDebug, H256};
 
 use crate::ceremonies::ProofOfAttendance;
-use crate::communities::CommunityIdentifier;
 use crate::scheduler::CeremonyIndexType;
 
-pub type ProofOfPersonhoodRequest<Signature, AccountId> =
-    Vec<(CommunityIdentifier, ProofOfAttendance<Signature, AccountId>)>;
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Default, RuntimeDebug)]
+pub struct IssueProofOfPersonhoodConfidenceCall<Signature, AccountId> {
+    call_index: [u8; 2],
+    requester: AccountId,
+    request: Vec<ProofOfAttendance<Signature, AccountId>>,
```

Comment:
ProofOfAttendance should be opaque to the faucet parachain. Please replace this with a `<Vec<u8>`
For cantillon, the payload will be encrypted anyway

Reply:
agreed